src/components/ActionsMenu/Item/ActionsMenuItem.st.css

```diff
@@ -0,0 +1,73 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+
+:vars {
+    /*Defaults*/
+    DefaultItemFont: 16px Avenir;
+    DefaultItemSubtitleFont: 14px Avenir;
+}
+
+:vars {
+    /*Overrides*/
+    ItemFont:  --overridable;
+    ItemSubtitleFont:  --overridable;
+}
+
+:vars {
+    itemFont: color(fallback(value(ItemFont), value(DefaultItemFont)));
+    itemSubtitleFont: fallback(value(ItemSubtitleFont), value(DefaultItemSubtitleFont));
+}
+
+.root {
+    -st-states: withIcon, disabled;
+    height: 40px;
+    padding: 8px 24px;
+    display: flex;
+    width: 100%;
+    cursor: pointer;
+}
+
+.root:withIcon {
+
+}
+
+.root:mobile {
+    width: 100%;
+}
+
+.root:hover {
+    background-color: rgba(0,0,0, 0.06);
+}
+
+.root:disabled {
+    opacity: 0.3;
+}
+
+.root:disabled:hover {
+    background-color: inherit;
```

Comment:
I think you should also set `cursor: not-allowed`

Reply:
@giladsegal i'm not sure changing the cursor is necessary.
I don't see it in the spec.
But we can check with the designers

Reply:
@jonathanadler any updates on this?

Reply:
No need to change cursor